### ISSUE TITLE
Uncaught TypeError: axe.run arguments are invalid

### ISSUE THREAD

**Issue by Glinkis** [angeloashmore/gatsby-plugin-react-axe#8]
I'm getting an error when I add `gatsby-plugin-react-axe` to my plugins array.

```
axe.js:10300 Uncaught TypeError: axe.run arguments are invalid
    at normalizeRunParams (axe.js:10300)
    at Object../node_modules/axe-core/axe.js.axe.run (axe.js:10330)
    at requestIdleCallback.timeout (index.js:160)
    at runTasks (index.js:138)
```

**Comment by leepowelldev** [comment]
Getting the same error

**Comment by angeloashmore** [comment]
Hi, thanks for the report. Do either of you mind submitting a PR to address the issue?

**Comment by leepowelldev** [comment]
I'm now investigating what may be causing it.

**Comment by leepowelldev** [comment]
Done. PR available.

**Comment by angeloashmore** [comment]
Thank you @Glinkis and @leepowelldev for opening the issue and submitting the PR.

The fix is merged and published as `v0.4.1`.
